### PR TITLE
Added instructions for running local xmtp server/unit tests locally

### PR DESCRIPTION
**Pull request by cameronvoell**
See updated README.md and new [example/README.md](https://github.com/xmtp/xmtp-react-native/blob/cameronvoell/add-local-unit-test-readme/example/README.md). 

My thinking here is to get commands used for running and testing the Example app explicit stated in the repo. Feedback welcomed!

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+To use the example app, run:
+
+```bash
+npm install
+cd example
+npm install --force
+npm run [ios or android]
+```
+
+## Running Example App Unit tests on local emulators
```
Comment (alexrisch) [inline example/README.md]: 🔥

**Review by alexrisch — APPROVED** [review]
Looks good to me!

File: example/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# XMTP React Native Example App
```
Comment (jhaaaa) [inline example/README.md]: Just a few suggested tech writerly style edits 🤓

```suggestion
# XMTP React Native example app
```

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+This basic messaging app has an intentionally unopinionated UI to help make it easier for you to build with.
+
+## Running the Example App
```
Comment (jhaaaa) [inline example/README.md]:
```suggestion
## Run the example app
```

File: example/README.md (hunk 1 above)
Comment (jhaaaa) [inline example/README.md]:
```suggestion
## Run example app unit tests on local emulators
```

File: example/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+npm install
+cd example
+npm install --force
+npm run [ios or android]
+```
+
+## Running Example App Unit tests on local emulators
+Running tests locally is useful when updating GitHub actions, or locally testing between changes.
+
+1. [Install docker](https://docs.docker.com/get-docker/)
```
Comment (jhaaaa) [inline example/README.md]:
```suggestion
1. [Install Docker](https://docs.docker.com/get-docker/)
```

File: example/README.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    docker-compose -p xmtp -f dev/local/docker-compose.yml up -d
+    ```
+3. Verify XMTP server is running
+    ```bash
+    docker-compose ls
+
+    NAME                STATUS              CONFIG FILES
+    xmtp                running(3)          <REPO_DIRECTORY>/xmtp-react-native/example/dev/local/docker-compose.yml
+    ```
+4. You can now run Unit tests on your local emulators
```
Comment (jhaaaa) [inline example/README.md]:
```suggestion
4. You can now run unit tests on your local emulators
```

File: example/README.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+5. You can stop the local xmtp server with the following command
```
Comment (jhaaaa) [inline example/README.md]:
```suggestion
5. You can stop the XMTP server with the following command:
```

File: example/README.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+## Running Example App Unit tests on local emulators
+Running tests locally is useful when updating GitHub actions, or locally testing between changes.
+
+1. [Install docker](https://docs.docker.com/get-docker/)
+
+2. Start a local XMTP server (from example directory)
+    ```bash
+    docker-compose -p xmtp -f dev/local/docker-compose.yml up -d
+    ```
+3. Verify XMTP server is running
```
Comment (jhaaaa) [inline example/README.md]:
```suggestion
3. Verify the XMTP server is running
```

**Review by jhaaaa — APPROVED** [review]
Thank you for taking the time to provide these meaningful docs, @cameronvoell! Just added a few suggested tech writer-type style edits 🤓

**Comment by cameronvoell** [comment]
> Thank you for taking the time to provide these meaningful docs, @cameronvoell! Just added a few suggested tech writer-type style edits 🤓

@jhaaaa thanks so much, applied your edits (very helpful!), and will go ahead and merge now 🙇

**Pull request merged**